tempesta_fw/http.c

```diff
@@ -391,6 +407,7 @@ tfw_http_prep_redirect(TfwHttpMsg *resp, unsigned short status, TfwStr *cookie,
 	static TfwStr c_len_crlf = {
 		.ptr = S_REDIR_C_LEN, .len = SLEN(S_REDIR_C_LEN) };
 	TfwStr host, *rh, *cookie_crlf = &crlf, *r_end;
+	DEFINE_TFW_STR_PROTO(proto, req->conn);
```

Comment:
It's also possible to create statically defined array `proto[2]` to contain both `http://` and `https://`, and use `TFW_CONN_PROTO(conn) == TFW_FSM_HTTPS` as the array index.

Reply:
Corrected.